lmnet/lmnet/visualize.py

```diff
@@ -129,6 +129,53 @@ def visualize_semantic_segmentation(image, post_processed, config):
     return result
 
 
+def visualize_keypoint_detection(image, joints, original_image_size=None):
+    """Draw keypoint detection result joints to image.
+
+    Args:
+        image: a numpy array of shape (height, width, 3).
+        joints: a numpy array of shape (num_joints, 3).
+        original_image_size: a tuple, (original_height, original_width). If not None, joints will be scaled.
+
+    Returns:
+        drawed_image: a numpy array of shape (height, width, 3).
+
+    """
+
+    if original_image_size is not None:
+        height, width, _ = image.shape
+        scale_height = height / original_image_size[0]
+        scale_width = width / original_image_size[1]
+        joints[:, 0] *= scale_width
+        joints[:, 1] *= scale_height
+
+    image = PIL.Image.fromarray(image, mode="RGB")
+    draw = PIL.ImageDraw.Draw(image)
+
+    num_joints = joints.shape[0]
+
+    joint_pairs = [[0, 1], [1, 3], [0, 2], [2, 4],
```

Comment:
Need comment for this list.

Reply:
@iizukak Thanks. I have added comments for this line.